docs/0.4/examples/gameroom/walkthrough/index.md

```diff
@@ -563,9 +581,13 @@ REST API asks the Gameroom UI to sign it with Alice's information.
     authorization_type: Trust
     members:
       - node_id: acme-node-000
-        endpoint: tls://splinterd-node-acme:8044
+        endpoints: [
+          - tls://splinterd-node-acme:8044
```

Comment:
mentions of tls:// should be changed to tcps://

Reply:
Updated!